packages/alert/src/AlertProvider.tsx

```diff
@@ -0,0 +1,71 @@
+/* istanbul ignore file */
+import React, { createContext, useContext, useState } from 'react'
+
+import Alert, { AlertProps } from './Alert'
+
+type AlertContextState = {
+  toast: (params: AlertProps) => void
```

Comment:
Since this is a function, can we rename this to `addToast` or something similar that makes it obvious that it performs an action?